guides/common/modules/ref_supported-host-operating-systems-and-architectures-in-registration.adoc

```diff
@@ -0,0 +1,25 @@
+[id="supported-host-operating-systems-and-architectures-in-registration_{context}"]
+= Supported Host Operating Systems and Architectures in Registration
+
+{Project} supports the following operating systems and architectures for host registration:
+
+Supported Host Operating Systems::
+
+Hosts must use one of the following {EL} versions:
```

Comment:
This is "satellite-only."

For Foreman: See list at the bottom: https://theforeman.org/introduction.html

For orcharhino:

Alma Linux
Amazon Linux
CentOS
Debian
Oracle Linux
Red Hat Enterprise Linux
Rocky Linux
SUSE Linux Enterprise Server
Ubuntu

*Architectures*: For Foreman, I think there's only a client for x86_64/amd64. For orcharhino, there are orcharhino Clients for aarch64 for Debian10/11, SLES 15, Ubuntu 20.04/22.04; ppc64le for Debian 10 and SLES 15; and x86_64/amd64 for all OS.

Reply:
Thanks!